spark-doris-connector/src/main/java/org/apache/doris/spark/DorisStreamLoad.java

```diff
@@ -162,8 +120,10 @@ private HttpPut getHttpPut(String label, String loadUrlStr) {
                     .filter(entry -> !"read_json_by_line".equals(entry.getKey()))
                     .forEach(entry -> httpPut.setHeader(entry.getKey(), entry.getValue()));
         }
-        if (fileType.equals("json")) {
+        if (dataFormat.getType().equalsIgnoreCase(FormatEnum.json.name())) {
             httpPut.setHeader("strip_outer_array", "true");
+            //to solve the error : The size of this batch exceed the max size [104857600]  of json type data. Split the file, or use 'read_json_by_line'
+            httpPut.setHeader("read_json_by_line","true");
```

Comment:
The data format required by read_json_by_line is {},{},{}, but the serialization in spark is [{},{},{}]

Reply:
I read https://doris.apache.org/zh-CN/docs/data-operate/import/import-way/load-json-format/ this document, I already know the difference between the two, It may indeed cause some problems, although I haven't found any problems in my current tests, I will remove this read_json_by_line configuration in json format first, and consider optimizing it in the next PR，thank you!!!

Reply:
I have removed the "read_json_by_line","true" configuration. But I'm confused，I have been using this configura to solve the [The size of this batch exceed the max size] problem for a long time and find nothing wrong。Maybe I need to study the code of be. Thank you！

Reply:
The default maximum input of jsonarray is 100MB (be.conf: streaming_load_json_max_mb ), maybe you can add a maximum byte limit

Reply:
Maybe I check the size of the data, if it is larger than 100MB, split it and write the split data in blocks? @JNSimba

Reply:
Yes, but it is better to add a parameter to control the value of this size

Reply:
OK，thanks for your advice，I will improve it!